cdc/kv/client.go

```diff
@@ -961,7 +959,8 @@ func (s *eventFeedSession) divideAndSendEventFeedToRegions(
 				zap.Reflect("regions", metas),
 				zap.String("changefeed", s.client.changefeed))
 			return nil
-		}, retry.WithBackoffMaxDelay(50), retry.WithMaxTries(100), retry.WithIsRetryableErr(cerror.IsRetryableError))
```

Comment:
it looks that this is the root cause of the problem.

how about add some comment about retry logic here.

Reply:
Comments added in kv client config.